dev.py

```diff
@@ -355,7 +358,9 @@ def parse_args(self):
         )
 
         # Build command
-        build_parser = subparsers.add_parser("build", help="Build the project")
+        build_parser = subparsers.add_parser(
+            "build", help="Build the project", aliases=["install"]
```

Comment:
```suggestion
            "build", help="Build and install the project", aliases=["install"]
```
This doc update improves the clarity